editor/header/preview-button/index.js

```diff
@@ -74,15 +74,29 @@ export class PreviewButton extends Component {
 			this.getWindowTarget()
 		);
 
-		const popupLoader = renderToString(
+		const popupLoader = `
 			<div>
 				<p>Please wait&hellip;</p>
 				<p>Generating preview.</p>
-			</div>
-		);
-		const css = '<style type="text/css"> div { margin-top: 25%; } p { text-align: center; } </style>';
+			</div>`;
+		const css = `
```

Comment:
Should we just combine these two into a single variable of markup?

Reply:
That figures.